src/scene/scene_forwardrenderer.js

```diff
@@ -631,7 +636,31 @@ pc.extend(pc, function () {
             drawCalls.sort(sortDrawCalls);
 
             // Render a depth target if the camera has one assigned
-            if (camera._depthTarget) {
+            if (camera._renderDepthRequests) {

```

Comment:
Fair point, but we should be trying to make the code better :-)

In this case, shouldn't we just have a method which is `camera.hasDepthMap` or whatever it should be called?